creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -1001,13 +1014,26 @@ class SnapbackSM {
     throw new Error(`Secondary ${secondaryUrl} did not sync up to primary for user ${wallet} within ${timeoutMs}ms`)
   }
 
-  async createEndpointToSpIdMap () {
+  /**
+   * Updates `this.endpointToSPIdMap` to the mapping of <endpoint : spId>. If the fetch fails, rely on the previous
+   * `this.endpointToSPIdMap` value. If the previous value is also an empty object, disable reconfig.
+   */
+  async updateEndpointToSpIdMap () {
     let endpointToSPIdMap = {}
-    const contentNodes = await this.audiusLibs.ethContracts.getServiceProviderList('content-node')
-    contentNodes.forEach(cn => { endpointToSPIdMap[cn.endpoint] = cn.spID })
+    try {
+      const contentNodes = await this.audiusLibs.ethContracts.getServiceProviderList('content-node')
+      contentNodes.forEach(cn => { endpointToSPIdMap[cn.endpoint] = cn.spID })
+    } catch (e) {
+      this.logError(`[updateEndpointToSpIdMap]: ${e.message}`)
+    }
 
-    if (Object.keys(endpointToSPIdMap).length === 0) throw new Error('Unable to initialize this.endpointToSPIdMap')
-    return endpointToSPIdMap
+    if (Object.keys(endpointToSPIdMap).length > 0) this.endpointToSPIdMap = endpointToSPIdMap
+    if (Object.keys(this.endpointToSPIdMap).length === 0) {
```

Comment:
approved anyway, if you don't want to change this its fine

Reply:
I didn't do ^ bc in the case `endpointToSPIdMap` is empty, this would default `this.snapbackReconfigEnabled = false` rather than using the backup version of `this.endpointToSPIdMap` if the map is not empty

Reply:
ah i see, didn't see the distinction between this.map and map. sounds good!